src/OpenTelemetry.Exporter.Prometheus/README.md

```diff
@@ -11,8 +11,18 @@
 
 ### Step 1: Install Package
 
+Depending on the hosting mechanism you will be using:
```

Comment:
This is not clear..  We should document the usage under respective packages. (I left a qn earlier about the future of OpenTelemetry.Exporter.Prometheus package as well, that is related to this)